opendevin/_logging.py

```diff
@@ -0,0 +1,121 @@
+import logging, os, sys
+from time import strftime
+import traceback
+
+console_formatter = logging.Formatter(
+    "\033[92m%(asctime)s - %(name)s:%(levelname)s\033[0m: %(filename)s:%(lineno)s - %(message)s",
+    datefmt="%H:%M:%S",
+)
+file_formatter = logging.Formatter(
+    "%(asctime)s - %(name)s:%(levelname)s: %(filename)s:%(lineno)s - %(message)s",
+    datefmt="%H:%M:%S",
+)
+llm_formatter = logging.Formatter(
+    "%(message)s"
+)
+
+def get_console_handler():
+    """
+    Returns a console handler for logging.
+    """
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(logging.INFO)
+    console_handler.setFormatter(console_formatter)
+    return console_handler
+
+def get_file_handler():
+    """
+    Returns a file handler for logging.
+    """
+    log_dir = os.path.join(os.getcwd(), "logs")
+    os.makedirs(log_dir, exist_ok=True)
+    file_handler = logging.FileHandler(os.path.join(log_dir, "opendevin.log"))
```

Comment:
```suggestion
    timestamp = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
    file_name = f"opendevin_{timestamp}.log"
    file_handler = logging.FileHandler(os.path.join(log_dir, file_name ))
```

Adding the timestamp would be better just for visibility and following the changes over each execution, what do you think?

Reply:
Absolutely, thank you!

Reply:
I guess you should `import datetime`